fuzz/fuzz_targets/block.rs

```diff
@@ -28,28 +28,20 @@ const MEM_SIZE: u64 = 256 * 1024 * 1024;
 const QUEUE_SIZE: u16 = 16; // Max entries in the queue.
 
 fuzz_target!(|bytes| {
-    if bytes.len() as u64 > MEM_SIZE {
+    if bytes.len() < 28 {
         return;
     }
 
-    let mem = GuestMemoryMmap::from_ranges(&[(GuestAddress(0), MEM_SIZE as usize)]).unwrap();
-    if mem.write_slice(bytes, GuestAddress(0 as u64)).is_err() {
+    let queue_data = &bytes[..28];
+    let mem_bytes = &bytes[28..];
+    if mem_bytes.len() as u64 > MEM_SIZE {
```

Comment:
You could have combined both checks together:
```rust
if bytes.len() < 28 + MEM_SIZE {
```

Reply:
Good inputs. In addition to what you suggested, I also added a new const `QUEUE_DATA_SIZE` to make it read a bit cleaner.